blocks/author-bio-block/README.md

```diff
@@ -21,9 +15,24 @@ The Author Bio Block expects the `credits` section from Composer's ANS schema, w
 ```
 credits: {
     by: [{
-        name: ...
-        url: ...
-        ...
+		type: "author"
+        name: "author_name"
+        description: "description",
+	image: {
+		url: "img_url",
+	},
+	social_links: [
+		{
+			{ site: "twitter", url: "https://twitter.com/sLcarothers" },
+
+		}
+	],
+	resized_params: {
```

Comment:
resized params aren't used any more. these can be removed in the readme

Reply:
removed it.

Reply:
nice